ethereum/contracts/DOTApp.sol

```diff
@@ -72,26 +73,29 @@ contract DOTApp is FeeController, AccessControl {
                 _channelId == ChannelId.Incentivized,
             "Invalid channel ID"
         );
-        token.burn(msg.sender, _amount, abi.encodePacked(_recipient));
+        token.burn(msg.sender, _amount);
 
         OutboundChannel channel =
             OutboundChannel(channels[_channelId].outbound);
 
         bytes memory call = encodeCall(msg.sender, _recipient, _amount);
         channel.submit(msg.sender, call);
+
+        emit Burned(msg.sender, _recipient, _amount);
     }
 
     function mint(
         bytes32 _sender,
         address _recipient,
         uint256 _amount
     ) external onlyRole(INBOUND_CHANNEL_ROLE) {
-        token.mint(_recipient, _amount, abi.encodePacked(_sender));
+        token.mint(_recipient, _amount);
+        emit Minted(_sender, _recipient, _amount);
     }
 
     // Incentivized channel calls this to charge (burn) fees
     function handleFee(address feePayer, uint256 _amount) external override onlyRole(FEE_BURNER_ROLE) {
-        token.burn(feePayer, _amount, "");
+        token.burn(feePayer, _amount);
```

Comment:
The ERC777 code would emit a `Burned` event here. Do we need to also emit a burned event?

Reply:
Nope. Nothing is using that event.